Cargo.toml

```diff
@@ -160,7 +160,7 @@ alloy-eips = { version = "0.4.2", default-features = false }
 alloy-consensus = { version = "0.4.2", default-features = false, features = ["serde", "serde-bincode-compat"] }
 alloy-network = { version = "0.4.2", default-features = false }
 
-citrea-e2e = { git = "https://github.com/chainwayxyz/citrea-e2e", rev = "7b392a2" }
+citrea-e2e = { git = "https://github.com/chainwayxyz/citrea-e2e", branch = "rakanalh/max-l2-blocks" }
```

Comment:
We have to set the revision here once https://github.com/chainwayxyz/citrea-e2e/pull/84 is merged